logstash-core/lib/logstash/api/lib/app/command.rb

```diff
@@ -23,7 +23,7 @@ def uptime
     end
 
     def started_at
-      (service.agent.started_at.to_f*1000.0).to_i
+      (LogStash::Agent::STARTED_AT.to_f * 1000.0).to_i
```

Comment:
I don't think its needed, its not hit really often. I don't believe we need this optimization